README.md

```diff
@@ -59,6 +59,16 @@ zmqpubrawblock=tcp://127.0.0.1:29000
 zmqpubrawtx=tcp://127.0.0.1:29000
 ```
 
+Depending on the actual hardware configuration, it may be useful to provide increased `dbcache` parameter value for faster verification and `rpcthreads` parameter value for better handling of multiple simultaneous API requests from `bitcoind` side. 
+
+```conf
+dbcache=2048
+rpcworkqueue=128
```

Comment:
nit: it's helpful to add small comments to explain what each configuration option does, since these are somewhat advanced tweaks.

```suggestion
# UTXO database cache size, in MiB
dbcache=2048
# Number of allowed pending RPC requests (default is 16)
rpcworkqueue=128
```